src/components/AddOrRemoveLightBox.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import LightBox from "./LightBox";
+// import LightBox from './components/LightBox'
+
+class AddOrRemoveLightBox extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            lightsArray: []
+        }
+    }
+    //create a addbox function, whenever it's got called, it's going to add a lightbox
+    addBox = () => {
+        //use spread operater without considering the lightarray is empty or not
+        this.setState({ lightsArray: [...this.state.lightsArray, <LightBox />] })
+        // alert("adding")
+    }
+    //create a removebox function, whenever it's got called, it's going to remove a lightbox
+    removeBox = (index) => {
+        //use if-else statement to check the array is null or not,if the array is not empty 
+        if (this.state.lightsArray[0] != null) {
+            //alert("removing")
```

Comment:
Excellent use of pseudo code here. I really like how you used alerts to guide you through your process.